operator/v1/types_dns.go

```diff
@@ -376,6 +409,20 @@ const (
 	NetworkResolverType   UpstreamType = "Network"
 )
 
+// ProtocolStrategy is a preference for the protocol to use for DNS queries.
+// +kubebuilder:validation:Enum:=TCP;""
+type ProtocolStrategy string
+
+var (
+	// ProtocolStrategyDefault specifies no opinion for DNS protocol.
+	// Currently, this means that CoreDNS uses the protocol of the originating client request as the upstream protocol.
+	// Note that the default is subject to change.
```

Comment:
Meaning the default is "CoreDNS uses the protocol of the originating client request as the upstream protocol". Which is the subject to change?

Reply:
Correct. I've updated this comment to be more explicit as well.